src/tools/utils/utils.js

```diff
@@ -0,0 +1,44 @@
+const { execSync } = require('child_process');
+const path = require('path')
+
+function getCommitDetails(commitSha) {
+    const commitDetails = execSync(`git show -s --format="%H;%an;%s" ${commitSha}`).toString().trim();
+    const [sha, authorName, subject] = commitDetails.split(';');
+    return { sha, authorName, subject };
+  }
+  
+  function getBranchName() {
+    return execSync('git rev-parse --abbrev-ref HEAD').toString().trim();
+  }
+  
+  function getUserAvatar() {
+    return "https://i.imgur.com/lSCFcFQ.gif"
+  }
+  
+  function getLastCommitFiles(commitSha) {
+    return execSync(`git diff-tree --no-commit-id --name-only -r ${commitSha}`)
+    .toString()
+    .trim()
+    .split('\n');
+  }
+  
+  function isImage(file) {
+    const imageExtensions = ['.jpg', '.jpeg', '.png', '.gif', '.bmp', '.svg'];
+    return imageExtensions.includes(path.extname(file).toLowerCase());
+  }
+  
+  function formatName(filePath) {
+    // examples: "> (com.br) 14XR1.png" and "> acc_chest_anubisbackpack.png"
```

Comment:
Not a big fan of this one
I prefer to just include the path and it look like this
Badge : badges/de/ITF05
Other : furnis/circus_c24_bluetentcrackable
Correct me if i miss understand the function

Reply:
It was like this before, but was too ambigious since theres only 1 channel instead of 5
And idk personnally i prefer, its more clear you know

Reply:
Mhh maybe :
> Clothe: xxxx
> Badge (com.br): xxx
what do you think ?

Reply:
mhh nah should be fine